### PR TITLE
admin replicate update: rename --sync flag to --mode

### PR DESCRIPTION
**Pull request by poornas**
## Motivation and Context
ui 

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — CHANGES_REQUESTED** [review]
We still have to preserve previous flag for few releases and deprecate it later

File: cmd/admin-replicate-update.go [hunk 1]
```diff
@@ -39,9 +39,9 @@ var adminReplicateUpdateFlags = []cli.Flag{
 		Usage: "endpoint for the site",
 	},
 	cli.StringFlag{
-		Name:  "sync",
-		Usage: "enable synchronous replication for this target, valid values are ['enable', 'disable'].",
-		Value: "disable",
+		Name:  "mode",
```
Comment (vadmeste) [inline cmd/admin-replicate-update.go:42]: I think you can add 'sync' to Aliases field, to keep backward compatibility

Comment (poornas) [reply cmd/admin-replicate-update.go:42]: well, the field changed to take on different values - so will have to validate this separately

**Review by harshavardhana — APPROVED** [review]

**Review by shtripat — APPROVED** [review]
lgtm, verified

**Pull request merged**